### DISCUSSION TITLE
Alguien con audio entre cortado cuando tienes 2 rpi funcionando?

### DISCUSSION THREAD

**Discussion by gestcodev** [mikebrady/shairport-sync#1573] [Q&A]
Hola, Tengo el siguiente escenario. 

Tengo una raspberri 4 y una 3+ y cuando transmito audio desde el iPhone se retrasa, se corta y luego se cae y vuelve a funcionar. Alguein sabe como poder arreglarlo? 

Hello, I have the following scenario.

I have a raspberri 4 and a 3+ and when I stream audio from the iPhone it lags, cuts off and then drops and works again. Anyone know how to fix it?

**Comment by mikebrady** [comment]
Thanks. Please see the [TROUBLESHOOTING.md](https://github.com/mikebrady/shairport-sync/blob/development/TROUBLESHOOTING.md) guide -- there might be something there. Or maybe some of the Issues, including closed ones. If you have no success, please open an Issue (not a Discussion).
